book/graphics.md

```diff
@@ -119,16 +119,15 @@ rectangular Tk widget that you can draw circles, lines, and text
 on. For example, you can create a canvas with Tk like this:[^canvas]
 
 [^canvas]: You may be familiar with the HTML `<canvas>` element, which is a
-    similar idea: a two-dimensional (2D) rectangle in which you can draw shapes.
+    similar idea: a two-dimensional rectangle in which you can draw shapes.
 
 ``` {.python .example}
-WIDTH, HEIGHT = 800, 600
 window = tkinter.Tk()
-canvas = tkinter.Canvas(window, width=WIDTH, height=HEIGHT)
+canvas = tkinter.Canvas(window, width=800, height=600)
 canvas.pack()
```

Comment:
The `WIDTH/HEIGHT` line is not part of the example, it's actually part of the browser. So I moved it to the next code block that's actually part of the browser.